<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>student</class>
 <widget class="QDialog" name="student">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1191</width>
    <height>703</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion Etudiants</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{

background-color:transparent;
color: #aa54ff; 
}



#centralwidget{
background-color: #040f13;
}
#sidemenu{
background-color: #071e26; 
border-radius: 20px; 
}
#ajouter 
{
padding: 20px; 
background-color: #7f3fbf; 
border-radius: 5px;
color : #fff;
}
QPushButton,QLineEdit{

background-color: #040f13; 
border-radius: 5px; }

#header{
background-color: #071e26; 
border-radius: 10px;
}
#rightsidemenu
{
background-color: #071e26; 
border-radius: 20px; 
}

#mainbody{
background-color: #071e26; 
border-radius: 10px;
}</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>1171</width>
     <height>691</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>86</y>
      <width>1021</width>
      <height>601</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QWidget" name="mainbody" native="true">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>0</y>
       <width>751</width>
       <height>601</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>50</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>100</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>0</y>
        <width>281</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Bahnschrift Light Condensed</family>
        <pointsize>31</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Liste des étudiants</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <property name="indent">
       <number>-1</number>
      </property>
     </widget>
     <widget class="QLineEdit" name="recherche">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>80</y>
        <width>441</width>
        <height>31</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Rechercher un étudiant</string>
      </property>
     </widget>
     <widget class="QTableView" name="tab_etudiant">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>130</y>
        <width>721</width>
        <height>261</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_supprimer">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>480</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Supprimer avec ID</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_supprimer">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>520</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>supprimer</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_tri_asc">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>50</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>ID sort</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_tri_desc">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>90</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Name sort</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="email">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>440</y>
        <width>441</width>
        <height>31</height>
       </rect>
      </property>
      <property name="inputMask">
       <string/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Normal</enum>
      </property>
      <property name="placeholderText">
       <string>taper l'email ici</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="textemail">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>490</y>
        <width>351</width>
        <height>71</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>input email text here</string>
      </property>
     </widget>
     <widget class="QPushButton" name="send">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>490</y>
        <width>81</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>envoyer</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_recherche">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>80</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_7">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>440</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Buzzer ON</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_8">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>470</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Buzzer OFF</string>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="rightsidemenu">
     <property name="geometry">
      <rect>
       <x>780</x>
       <y>0</y>
       <width>221</width>
       <height>601</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>61</x>
        <y>450</y>
        <width>20</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>+</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>20</y>
        <width>141</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe Fluent Icons</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Ajouter /Modifier</string>
      </property>
     </widget>
     <widget class="QFrame" name="ajout">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>181</width>
        <height>371</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLineEdit" name="le_id">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>161</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>Id Etudiant</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_nom">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>74</y>
         <width>161</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>Nom</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_prenom">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>118</y>
         <width>161</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>Prenom</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_email">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>162</y>
         <width>161</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>Email</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_specialite">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>206</y>
         <width>161</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>Specialite</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_mention">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>250</y>
         <width>161</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>Mention</string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="pb_modifier">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>500</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe Fluent Icons</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Modifier</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_ajouter">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>430</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe Fluent Icons</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Ajouter</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QFrame" name="header">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1171</width>
      <height>81</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>50</height>
     </size>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QFrame" name="frame_8">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>9</y>
       <width>72</width>
       <height>32</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>20</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>40</width>
       <height>30</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_7"/>
    </widget>
    <widget class="QPushButton" name="pb_menu">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>121</width>
       <height>51</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>50</horstretch>
       <verstretch>10</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
     <property name="text">
      <string>Menu</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>icons/hamburger.png</normaloff>icons/hamburger.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>16</width>
       <height>20</height>
      </size>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../../Téléchargements/icons/icons/hamburger.png</pixmap>
     </property>
    </widget>
    <widget class="QLabel" name="label_17">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>0</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>50</horstretch>
       <verstretch>50</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>50</width>
       <height>1</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="lineWidth">
      <number>2</number>
     </property>
     <property name="text">
      <string>InteLab</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_18">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>20</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../../Téléchargements/Downloads/notification.png</pixmap>
     </property>
    </widget>
    <widget class="QLabel" name="label_19">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>20</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../../Téléchargements/Downloads/video.png</pixmap>
     </property>
    </widget>
    <widget class="QLabel" name="label_24">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>40</y>
       <width>371</width>
       <height>41</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>50</horstretch>
       <verstretch>50</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>50</width>
       <height>1</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="lineWidth">
      <number>2</number>
     </property>
     <property name="text">
      <string>Gestion des étudiants</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>1040</x>
       <y>30</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Statistiques</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_pdf">
     <property name="geometry">
      <rect>
       <x>980</x>
       <y>30</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>pdf</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="sidemenu">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>90</y>
      <width>181</width>
      <height>601</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <widget class="QFrame" name="frame_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>161</width>
       <height>481</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>50</width>
       <height>0</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>141</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>  Encadrants</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>icons/teacher (1).png</normaloff>icons/teacher (1).png</iconset>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>141</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Etudiants</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>icons/student.png</normaloff>icons/student.png</iconset>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>170</y>
        <width>141</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Materiels</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>icons/flask.png</normaloff>icons/flask.png</iconset>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>240</y>
        <width>141</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Sponsor</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>icons/handshake.png</normaloff>icons/handshake.png</iconset>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>310</y>
        <width>141</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Publications</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>icons/research.png</normaloff>icons/research.png</iconset>
      </property>
     </widget>
     <zorder>pushButton_2</zorder>
     <zorder>pushButton_3</zorder>
     <zorder>pushButton_5</zorder>
     <zorder>pushButton_6</zorder>
     <zorder>pushButton_4</zorder>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
